t/README.md

```diff
@@ -0,0 +1,75 @@
+# Unit Tests for PG
+
+The individual unit tests are located in each of the directories.
+
+Formal unit tests are located in the the `macros` and `contexts` directories that are designed to test the pg macros and contexts respectively.
+
+## Running the tests
+
+```
+cd $PG_ROOT
+prove -r .
+```
+
+will run all of the tests in `.t` files within subdirectories of `t`.
+
+### Running an individual test
+
+If instead, you want to run an individual test, for example the `pgaux.t` test suite,
+```
```

Comment:
The language (`bash`) should be specified, and this line should be preceded with an empty line.